.github/workflows/test.yml

```
@@ -25,9 +17,34 @@ jobs:
+          compose-file: ./standalone-compose
+
+      - name: Run docker-compose for redis replication instance
+        uses: hoverkraft-tech/compose-action@v2.0.1
+        with:
+          compose-file: ./sentinel-compose
+
+      - name: Run docker-compose for redis cluster instance
+        uses: hoverkraft-tech/compose-action@v2.0.1
+        with:
+          compose-file: ./cluster-compose
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)